samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -7,24 +7,49 @@
 using Esri.ArcGISMapsSDK.Components;
 using Esri.HPFramework;
 using System.Collections;
-using System.Collections.Generic;
-using System.IO;
 using UnityEngine;
 using UnityEngine.Rendering;
 using UnityEngine.SceneManagement;
 using UnityEngine.UI;
 
 public class SampleSwitcher : MonoBehaviour
 {
-    public string APIKey = "";
-    public Dropdown PipelineTypeDropdown;
-    public Dropdown SceneDropdown;
+    public GameObject MenuVideo; 
+    private string APIKey;
     public Button ExitButton;
-    public List<string> SceneList = new List<string>();
-    public List<string> PipelineList = new List<string>();
+    private string PipelineModeText;
```

Comment:
In general, private member variables are in `lowerCamelCase`, so be sure to change all of these member names